systemtest/src/test/java/io/strimzi/systemtest/ConnectST.java

```diff
@@ -745,6 +752,142 @@ void testMultiNodeKafkaConnectWithConnectorCreation() {
         KafkaConnectUtils.waitForMessagesInKafkaConnectFileSink(connectorPodName, Constants.DEFAULT_SINK_FILE_PATH, "99");
     }
 
+    @Tag(EXTERNAL_CLIENTS_USED)
+    @Tag(CONNECTOR_OPERATOR)
+    @Test
+    void testConnectTlsAuthWithWeirdUserName() throws Exception {
+        // Create weird named user with . and maximum of 64 chars -> TLS
+        String weirdUserName = "jjglmahyijoambryleyxjjglmahy.ijoambryleyxjjglmahyijoambryleyxasd";
+        String connectClusterName = "connect-cluster";
+
+        KafkaResource.kafkaEphemeral(CLUSTER_NAME, 3)
+                .editSpec()
+                    .editKafka()
+                        .editListeners()
+                            .withNewKafkaListenerExternalNodePort()
+                            .endKafkaListenerExternalNodePort()
+                            .withNewTls()
+                            .endTls()
+                        .endListeners()
+                    .endKafka()
+                .endSpec()
+                .done();
+
+        KafkaUserResource.tlsUser(CLUSTER_NAME, weirdUserName).done();
+
+        // Initialize CertSecretSource with certificate and secret names for consumer
+        CertSecretSource certSecret = new CertSecretSource();
+        certSecret.setCertificate("ca.crt");
+        certSecret.setSecretName(KafkaResources.clusterCaCertificateSecretName(CLUSTER_NAME));
+
+        KafkaConnectResource.kafkaConnect(CLUSTER_NAME, 1)
+                .editMetadata()
+                    .addToLabels("type", "kafka-connect")
+                    .addToAnnotations(Annotations.STRIMZI_IO_USE_CONNECTOR_RESOURCES, "true")
+                .endMetadata()
+                .editSpec()
+                    .addToConfig("group.id", connectClusterName)
+                    .addToConfig("offset.storage.topic", connectClusterName + "-offsets")
+                    .addToConfig("config.storage.topic", connectClusterName + "-config")
+                    .addToConfig("status.storage.topic", connectClusterName + "-status")
+                    .withNewTls()
+                        .withTrustedCertificates(certSecret)
+                    .endTls()
+                    .withBootstrapServers(CLUSTER_NAME + "-kafka-bootstrap:9093")
+                .endSpec()
+                .done();
+
+        testConnectAuthorizationWithWeirdUserName(weirdUserName, SecurityProtocol.SSL);
```

Comment:
as far as I can see this will use an external client which will try to connect to the external node port listener via TLS but in the above configuration such external node port listener doesn't have TLS enabled. What am I missing?

Reply:
this is my fault, I'll take a look at it

Reply:
@ppatierno TLS is by default on true so I will not add that to external node port extra